rust/lance/src/index/vector/hnsw.rs

```diff
@@ -163,10 +163,11 @@ impl VectorIndex for HNSWIndex {
             )
         };
 
-        let ef = query.k + query.k / 2;
+        let k = query.k * query.refine_factor.unwrap_or(1) as usize;
```

Comment:
nit: do we want to introduce a new param for this?

Reply:
you mean `ef`?

Reply:
yeah, maybe we should just expose EF